### DISCUSSION TITLE
Is it safe to say that _only_ 500, 700, and 800 devices are supported right now?

### DISCUSSION THREAD

**Discussion by mdeneen** [zwave-js/zwave-js-ui#3158] [Q&A] [answered]
I have a 400 device and I'm wondering if it can be used with zwave-js.

**Comment by robertsLando** [comment]
I think it should work but better to ask to @AlCalzone

**Comment by AlCalzone** [comment, marked as answer]
In general, Z-Wave JS supports the older series aswell, since the protocol is backwards compatible. However some devices require polling for the status updates which isn't done automatically - so it isn't full out-of-the-box support. You may have to write an automation that refreshes the value regularly (but not too often to avoid network congestion).
